test/parallel/test-event-emitter-add-listeners.js

```diff
@@ -66,3 +66,10 @@ e1.on('hello', listen1);
 // The order of listeners on an event is not always the order in which the
 // listeners were added.
 assert.deepStrictEqual(e1.listeners('hello'), [listen2, listen1]);
+
+// Verify that the listener must be a function
+assert.throws(() => {
+  const ee = new EventEmitter();
+
+  ee.on('foo', null);
+}, /^TypeError: "listener" argument must be a function$/);
```

Comment:
What would you like the message to be? If an incorrect error is thrown, the message isn't used at all. If an error isn't thrown, it says `AssertionError: Missing expected exception.`, which is what I would have used as the message.